src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -170,6 +171,7 @@ public int GetHashCode(object obj)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal sealed class CultureAwareComparer : StringComparer
```

Comment:
We currently have no way to add these internal types into the mscorlib facade. We use the shipped .NET Framework mscorlib assembly and only type forward public types. We cannot even type forward these types to S.Private.CoreLib without internals visible to. I would just exclude these internal ones unless we know they are needed for a specific reason.